src/app/forms/templates/hgv/hgv-tech-record.template.ts

```diff
@@ -9,121 +15,150 @@ export const HgvTechRecord: FormNode = {
       name: 'vehicleType',
       label: 'Vehicle type',
       value: '',
-
+      width: FormNodeWidth.XS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.VEHICLETYPE
+      viewType: FormNodeViewTypes.VEHICLETYPE,
+      editType: FormNodeEditTypes.SELECT,
+      options: [
+        { value: 'psv', label: 'PSV' },
+        { value: 'hgv', label: 'HGV' },
+        { value: 'trl', label: 'Trailer' }
+      ],
     },
     {
       name: 'regnDate',
       label: 'Date of first registration',
       value: '',
-
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.DATE
+      viewType: FormNodeViewTypes.DATE,
+      editType: FormNodeEditTypes.DATE,
+      validators: [{ name: ValidatorNames.Required }],
+      isoDate: false
     },
     {
       name: 'manufactureYear',
       label: 'Year of manufacture',
       value: '',
-
+      width: FormNodeWidth.XS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      editType: FormNodeEditTypes.TEXT,
```

Comment:
Not sure whether we need to validate a year?

Reply:
Dont they want to change this anyway in the etl into nop ?